src/content/configuration/dev-server.md

```diff
@@ -742,12 +722,11 @@ module.exports = {
 };
 ```
 
-
 ## `devServer.noInfo` 🔑
 
 `boolean`
 
-Tells dev-server to supress messages like the webpack bundle information. Errors and warnings will still be shown. `devServer.noInfo` is disabled by default.
+Suppresses messages like the webpack bundle information. Errors and warnings will still be shown. `devServer.noInfo` is disabled by default.
```

Comment:
the previous version is better

Reply:
Alright. I’ll still update it to spell “suppress” correctly, though